src/inmanta/protocol/endpoints.py

```diff
@@ -357,13 +357,21 @@ def __init__(
         super().__init__(name)
         assert isinstance(timeout, int), "Timeout needs to be an integer value."
         LOGGER.debug("Start transport for client %s", self.name)
+        self.forced_instance = force_instance
         if with_rest_client:
             self._transport_instance = client.RESTClient(self, connection_timout=timeout, force_instance=force_instance)
         else:
             self._transport_instance = None
         self._version_match = version_match
         self._exact_version = exact_version
 
+    def close(self):
+        """
+        Closes the RESTclient instance manually. This is only needed when it is started with force_instance set to true
+        """
+        if self.forced_instance:
+            self._transport_instance.close()
```

Comment:
```suggestion
        self._transport_instance.close()
```
No need to have the condition on this level